desktop/core/src/desktop/js/parse/sql/sqlParserRepository.ts

```diff
@@ -48,7 +48,7 @@ const SYNTAX_MODULES = {
 };
 /* eslint-enable */
 
-class SqlParserRepository implements SqlParserProvider {
+export class SqlParserRepository implements SqlParserProvider {
```

Comment:
This should be singleton, hence the export default of the instance below. Although, the dynamic imports are likely resolve once, in which case the export below should be dropped.